client/components/boards/boardsList.jade

```diff
@@ -26,8 +26,6 @@ template(name="boardList")
 
 
 template(name="boardListHeaderBar")
-  h1 {{_ 'my-boards'}}
```

Comment:
iYes, sorry I didn’t refresh the diff view (when I said that you didn’t address my comment, I removed the message 1min after that when I realized that I was wrong)